src/constants/notificationConstants.js

```diff
@@ -25,3 +25,4 @@ export const COLLECTIBLE = 'COLLECTIBLE';
 export const CONNECTION = 'CONNECTION';
 export const SIGNAL = 'SIGNAL';
 export const BCX = 'BCX';
+export const BADGE = 'badge';
```

Comment:
Why lowercased?

Reply:
It is returned like this from the backend. Added a constant to match it

Reply:
👍